service/src/cli.yml

```diff
@@ -48,11 +54,6 @@ subcommands:
                 long: skip-ra
                 short: s
                 help: skip remote attestation. Set this flag if running enclave in SW mode
-            - w-server:
```

Comment:
Seemed to be obsolete, so I removed it.

Reply:
Hm.. I see. Then how about a renaming?  w-server isn't very descriptive. How about `external-worker-ip`?

Reply:
`external-address` seems fine to me. I'd use address here because it is generic and implies that we can use both: ip-addresses, and urls.

However, there is documentation that needs to be adjusted. So I wonder if you want to do this now.

Reply:
Readded it. But I didn't see any documentation.. not in the book and not in the worker.

Reply:
Ok, but @gaudenzkessler needs to know that he needs to update the flags in the docker demo

Reply:
nice!